src-ui/src/locale/messages.de_DE.xlf

```diff
@@ -4210,13 +4210,13 @@
         </context-group>
         <target state="final">Keine Elemente gefunden</target>
       </trans-unit>
-      <trans-unit id="6541407358060244620" datatype="html">
+      <trans-unit id="6541407358060244620" datatype="html" approved="yes">
         <source>Note: value has not yet been set and will not apply until explicitly changed</source>
         <context-group purpose="location">
           <context context-type="sourcefile">src/app/components/common/input/switch/switch.component.html</context>
           <context context-type="linenumber">45</context>
         </context-group>
-        <target state="needs-translation">Note: value has not yet been set and will not apply until explicitly changed</target>
+        <target state="final">Heinweis: der Wert wurde nicht gesetzt und wird nicht angewendet, bis er explizit geändert wurde</target>
```

Comment:
Changes should be made at crowdin

Reply:
Don't have a Crowdin account and it seems that "Login with github" requires a dedicated Crowdin account anyway, I don't want that ... if you have a Crowdin account, it would be great if you could correct this.

First word in line is wrong

```
"Heinweis:" > "Hinweis:"
  ^
 "e" is too much
```